### DISCUSSION TITLE
Add modified+created date to Pages in API responses

### DISCUSSION THREAD

**Discussion by nathanvogel** [GitbookIO/community#147] [Feature Requests]
### What's your feature/idea?

Hi, 
We would like to have the date on which a document was last modified returned by the API. This is in order to build workflows to compute content freshness and ensure all pages are maintained. 

### What's your desired outcome?

I would be able to sync metadata to our task tracking tool, in which we can manage document freshness and ownership.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

### Anything else?

I'm rating the feature request as medium, but could be a deal breaker in the long run. Maintainability of docs is one of our main concerns and GitBook doesn't support this with features like Archbee's [Document Verification](https://docs.archbee.com/document-verifications)

**Comment by outragedhuman** [comment]
We would like to see this too for the same reason, to build a tracking tool to prevent content from going stale without having to manually maintain Last Modified dates.

**Comment by taszap** [comment]
Hi @outragedhuman and @nathanvogel - we are working on some solutions to help writers identify stale content and while API with created/modified isn't exactly part of the scope just yet we hope to have some exciting updates on this front in the coming months

**Reply by outragedhuman** [reply]
Great, thanks!

**Comment by taszap** [comment]
Hi Both! 

We shipped a change to the API yesterday, which now includes CreatedAT and UpdatedAT in the responses for change requests and pages.

**Reply by outragedhuman** [reply]
Thank you so much! This looks like it's exactly what we needed, just working on hooking it up now. Much appreciate the quick turnaround.

**Reply by outragedhuman** [reply]
Implemented successfully, our tracking spreadsheet now has Last Modified conditionally formatted so we can track content freshness with ease. WOOO 🥳 

Note: "Page Not Found" for several entries because we're still building out content. 

![Screenshot 2023-11-01 at 4 37 04 PM](https://github.com/GitbookIO/community/assets/6742283/00963bf3-5491-4c44-9f2a-841173e7bf88)
